Airwallex/Core/Sources/AWXAPIClient.m

```diff
@@ -99,6 +111,12 @@ - (Class)responseClass {
     return nil;
 }
 
+- (NSString *)eventName {
+    NSString *requestName = NSStringFromClass([self class]);
```

Comment:
One concern I have about this is that this is an implicit enforcement on the naming of the class. If the class is named `AirwallexSomeRequest` then the Airwallex prefix won't be removed.

Reply:
Yeah this one is pretty hard, I don't have perfect solution to decide whether there is a prefix before the actual request content, unless we manually define the event name for each request which is bit trivial, but worse case scenario the event name will be `airwallex get payment methods`, which won't be too bad. And I've added comment in case in the future we will rename.